plugins/SitesManager/Tracker/SitesManagerRequestProcessor.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\SitesManager\Tracker;
+
+use Piwik\Date;
+use Piwik\Tracker\Cache;
+use Piwik\Tracker\Request;
+use Piwik\Tracker\RequestProcessor;
+use Piwik\Tracker\Visit\VisitProperties;
+use Piwik\Plugins\SitesManager\Model as SitesManagerModel;
+
+/**
+ * Handles site specific logic during tracking.
+ */
+class SitesManagerRequestProcessor extends RequestProcessor
+{
+    /**
+     * @var SitesManagerModel
+     */
+    private $sitesManagerModel;
+
+    public function __construct(SitesManagerModel $sitesManagerModel)
+    {
+        $this->sitesManagerModel = $sitesManagerModel;
+    }
+
+    public function recordLogs(VisitProperties $visitProperties, Request $request)
+    {
+        // if we successfully record some data and the date is older than the site's created time,
+        // update the created time so the data will be viewable in the UI
+        $visitorNotFoundInDb = $request->getMetadata('CoreHome', 'visitorNotFoundInDb');
```

Comment:
Started writing a test for this then realized that this only happens on an existing visit, and a visit in the past will always be a new visit. After the new visit, it could occur, but at this point the site's created date will be earlier then the next action time. So I will remove the check entirely :)